src/main/resources/db/changelog/v1.23.0/v1.23.0.yaml

```
@@ -24,3 +24,6 @@ databaseChangeLog:
             relativeToChangelogFile: true
             splitStatements: false
             endDelimiter: $_$;
+  - include:
+      relativeToChangelogFile: true
+      file: add-updated-at-feed-index.sql
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)